src/Symfony/Component/Form/Extension/Core/Type/TelType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class TelType extends AbstractType
```

Comment:
i definitely prefer `TelephoneType` here :thinking:

Reply:
Hello, I'd like to finish this PR, how should I name the class ? TelType, TelephoneType, PhoneType etc ? @symfony/deciders

Reply:
either way no blocker for me :) i understand we just want type="tel" one way or another :+1: (my reasoning still stands though(.

Reply:
I think we should go with types that match the html types because that's we we tend to do instead of trying to name them based of semantic of the field (`CheckboxType`, `TextareaType`, `RangeType` are good examples of this)

Reply:
@ro0NL I think your reasoning was wrong here when stating this (about "tel"): 

> we should not name our class after an implementation detail

HTML5 is exactly the opposite to an implementation detail: it's a public interface, a standard, the tip of the iceberg. So we should definitely not seek to be more clever than the guys who wrote it (we're not ;) ).